concert/cloudformation.yml

```diff
@@ -361,6 +361,40 @@ Resources:
         FunctionArn: !Ref OnEventStageChangedFunction.Alias
         RouteKey: !FindInMap [RouteKeys, EventStageChanged, Key]
 
+  OnStackChange:
```

Comment:
Can we move this below PrepareStackFunction, since it references it?

Reply:
Yep. Done.